tools/add_triage_signature.py

```diff
@@ -455,12 +542,57 @@ def get_jira_client(username, password):
     logger.info("log-in with username: %s", username)
     return jira.JIRA(JIRA_SERVER, basic_auth=(username, password))
 
+
 ############################
 # Signature runner functionality
 ############################
 LOGS_URL_FROM_DESCRIPTION_OLD = re.compile(r".*logs:\* \[(http.*)\]")
 LOGS_URL_FROM_DESCRIPTION_NEW = re.compile(r".*\* \[[lL]ogs\|(http.*)\]")
 
+
+def search_strings_in_file(file_name: str, substr: List[str]) -> List[str]:
+    with open(file_name, "r") as _file:
+        data = _file.read()
+
+    res = []
```

Comment:
maybe let's just yield here?

Reply:
Tried that and it returned me different outputs with the next `group_similar_strings` function. I'm not sure why :-\

Reply:
Function got refactored https://github.com/openshift-assisted/assisted-installer-deployment/pull/55#discussion_r598298233